eq-author/src/App/page/Design/Validation/GroupValidations/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import { get } from "lodash";
+import PropTypes from "prop-types";
+
+import SidebarButton, { Title, Detail } from "components/buttons/SidebarButton";
+import Button from "components/buttons/Button";
+import ModalDialog from "components/modals/ModalDialog";
+
+import TotalValidation from "./TotalValidation";
+
+import IconTotal from "./icon-calculator.svg?inline";
+
+export const TotalButton = styled(SidebarButton)`
+  display: flex;
+  align-items: center;
+  padding-right: 2em;
+`;
+
+const TotalIcon = styled(IconTotal)`
+  flex: 0 0 auto;
+`;
+
+const Details = styled.div`
+  margin-left: 0.1em;
+  flex: 1 1 auto;
+`;
+
+export const GroupValidationModal = styled(ModalDialog)`
+  .Modal {
+    width: 40em;
+    height: 23em;
+  }
+`;
+
+const Buttons = styled.div`
+  display: flex;
+  justify-content: flex-end;
+`;
+
+class GroupValidations extends Component {
+  static propTypes = {
+    totalValidation: PropTypes.shape({
+      entityType: PropTypes.string.isRequired,
+      custom: PropTypes.number,
+      previousAnswer: PropTypes.shape({
+        displayName: PropTypes.string.isRequired,
+      }),
+    }),
+    type: PropTypes.string,
+  };
+
+  state = { isModalOpen: false };
+
+  handleButtonClick = () => {
+    this.setState({ isModalOpen: true });
+  };
+  handleModalClose = () => {
+    this.setState({ isModalOpen: false });
+  };
+
+  render() {
+    const total = this.props.totalValidation;
+    return (
+      <>
+        <TotalButton
+          onClick={this.handleButtonClick}
+          data-test="sidebar-button-total-value"
+          disabled={!total}
+        >
+          <TotalIcon />
+          <Details>
+            <Title>Total</Title>
```

Comment:
This should display the "comparator". Eg. "Total less than or equal to"

Reply:
Would the symbol be enough?

Reply:
I have added the symbol in a bracket